config/initializers/simple_form.rb

```diff
@@ -66,7 +72,7 @@
   config.error_notification_tag = :div
 
   # CSS class to add for error notification helper.
-  config.error_notification_class = 'alert alert-error'
+  config.error_notification_class = 'error_notification'
```

Comment:
@AlexanderUlitin  Actually I don't see any difference...Everything seems to work fine